### DISCUSSION TITLE
SMapCountry use always a mercator shape ?

### DISCUSSION THREAD

**Discussion by EArminjon** [simplewidgets/countries_world_map#21] [General]
Just to know. When calling SMapIndia or SMapFrance, we always have a map 100% Mercator ?

**Comment by LauDijksterhuis** [comment]
@EArminjon I am not sure, when developing this package I just used available Maps online
So it is hard to say if the map is Mercator

Also not all maps have the GEO locations included.

**Reply by EArminjon** [reply]
Okay :'(

**Reply by EArminjon** [reply]
@LauDijksterhuis  from where did you get the map SVGs ? I'm curious

**Reply by LauDijksterhuis** [reply]
Hi @EArminjon ,

I used different sources for each map, when I find a SVG map, I throw it into a simple encoder app which I developed to create consistent strings with instructions and GEO box is possible. Not all maps would come out well which required me to either edit the SVG file or find a different map of the country that DOES convert well to the instructions string.

**Reply by EArminjon** [reply]
Oh i see, huge manual work. Ty for this info.

**Reply by tierecke** [reply]
> Hi @EArminjon ,
> 
> I used different sources for each map, when I find a SVG map, I throw it into a simple encoder app which I developed to create consistent strings with instructions and GEO box is possible. Not all maps would come out well which required me to either edit the SVG file or find a different map of the country that DOES convert well to the instructions string.

Is there a possibility to get the encoder? 
I'd be happy to revise some maps that change and maybe make new ones.

Thanks!
